using-valtimo/web/web.md

```diff
@@ -0,0 +1,9 @@
+# Web
```

Comment:
Can you also include a link to this page in `SUMMARY.md`? And a link to this page in `using-valtimo.md`?